### DISCUSSION TITLE
Path cannot be a privileged system path

### DISCUSSION THREAD

**Discussion by madcar86** [manyfold3d/manyfold#2784] [Support] [answered]
I recently redid my Manyfold container to start fresh. But now I keep getting the area "Path cannot be a privileged system path
" when I try to add a library. Everything should be the same as before. 

I am running docker on Promox with a mounted SMB share.

**Comment by Floppy** [comment]
Yep, just noticed this is a bug, my fault; #2786.

I'll get a fix out asap.

**Reply by Floppy** [reply, marked as answer]
OK, v0.80.1 is out and should fix this. Sorry!

https://github.com/manyfold3d/manyfold/releases/tag/v0.80.1

**Reply by madcar86** [reply]
Thanks. I'll redeploy and let you know.

**Reply by madcar86** [reply]
That did it. Thanks
